build/storage/tests/it/ptf_tests/system_tools/test_platform.py

```diff
@@ -96,62 +155,171 @@ def _install_docker(self):
     def check_system_setup(self):
         """Overwrite this method in specific platform if you don't want check all setup"""
         if not self._is_virtualization():
-            raise Exception("Virtualization is not setting properly")
+            raise DependenciesException("Virtualization is not setting properly")
```

Comment:
You always return "DependenciesException" although you defined specialized classes for the different cases - use them here.